pkg/random/random_test.go

```diff
@@ -20,3 +22,10 @@ func TestConcurrency(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestReader_Read(t *testing.T) {
+	var s []byte = make([]byte, 5)
+	NoBytesRead, err := Reader.Read(s)
```

Comment:
make variable NoBytesRead more descriptive, plus the first letter shouldn't be capitalized